sorc/link.sh

```diff
@@ -5,19 +5,30 @@ set -ex
 
```

Comment:
I don't think just `link.sh` is appropriate as it is too generic.
Please retain a suffix and ensure it is consistent with the `usage` blurb inside the script.

Reply:
Updated to link_workflow.sh